### ISSUE TITLE
iOS Keyboard for barter

### ISSUE THREAD

**Issue by Twjc2000** [alexbatalov/fallout2-ce#423]
Is there a way to bring up a keyboard when selecting the amount of an item to transfer to the trade tab? It's near impossible to use money unless you hold left mouse for 10 minutes straight. Someone else mentioned hackers keyboard but that's for Android and I haven't yet discovered a method to open a keyboard through the notifications or widget menus (currently using iOS 18).

**Comment by Twjc2000** [comment]
Just researched for awhile about force opening keyboards on ios, and it seems there isn’t a way. Maybe i’m not aware of how to activate a text field within the barter tab, but currently i can’t use money for bartering unless i can steal it back from the vendor.

**Comment by rjdavis9155-svg** [comment]
> Just researched for awhile about force opening keyboards on ios, and it seems there isn’t a way. Maybe i’m not aware of how to activate a text field within the barter tab, but currently i can’t use money for bartering unless i can steal it back from the vendor.
Yes this! I’m currently having the exact same problem. This is the only relevant site I can find about this. It’s beyond frustrating.

**Comment by phobos2077** [comment]
Does this happen in the [current fork](https://github.com/fallout2-ce/fallout2-ce)?

**Comment by menchhh1** [comment]
> Происходит ли это в [текущей версии игры](https://github.com/fallout2-ce/fallout2-ce) ?

same problem
